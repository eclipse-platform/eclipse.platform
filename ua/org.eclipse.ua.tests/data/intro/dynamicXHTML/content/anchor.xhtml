<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
       "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<title>anchor.xhtml</title>
</head>
<body>

<p>
<strong>The section below should have the entire contents of anchor2.xhtml.</strong>
</p>

<small>[begin anchor 1]</small>
<hr/>
<anchor id="anchor.1" />
<hr/>
<small>[end anchor 1]</small>

<p>
<strong>The section below should only have the second paragraph from anchor2.xhtml.</strong>
</p>

<small>[begin anchor 2]</small>
<hr/>
<anchor id="anchor.2" />
<hr/>
<small>[end anchor 2]</small>

</body>
</html>

