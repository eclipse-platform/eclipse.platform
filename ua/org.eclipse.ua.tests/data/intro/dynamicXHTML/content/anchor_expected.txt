<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>

<title>anchor.xhtml</title>
</head>
<body>

<p>
<strong>The section below should have the entire contents of anchor2.xhtml.</strong>
</p>

<small>[begin anchor 1]</small>
<hr/>
<p id="paragraph.1">
This is the first paragraph in anchor2.xhtml.
</p><p id="paragraph.2">
This is the second paragraph in anchor2.xhtml.
</p><p id="paragraph.3">
This is the third paragraph in anchor2.xhtml.
</p>
<hr/>
<small>[end anchor 1]</small>

<p>
<strong>The section below should only have the second paragraph from anchor2.xhtml.</strong>
</p>

<small>[begin anchor 2]</small>
<hr/>
<p id="paragraph.2">
This is the second paragraph in anchor2.xhtml.
</p>
<hr/>
<small>[end anchor 2]</small>

</body>
</html>